### ISSUE TITLE
Ubuntu 22.04 does not have Azure CLI 2.46.0 installed

### ISSUE THREAD

**Issue by oishii-code** [actions/runner-images#7277]
### Description

The Azure CLI bug fix for https://github.com/Azure/azure-cli/issues/25406 is not included in Ubuntu 22.04. The Azure CLI task is still using the previous version (2.45.0).

> Note The Windows 2022 (20230307.2) image has Azure CLI 2.46.0 installed.

### Platforms affected

- [X] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [X] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

Image: ubuntu-22.04
Version: 20230305.1
Included Software: https://github.com/actions/runner-images/blob/ubuntu22/20230305.1/images/linux/Ubuntu2204-Readme.md
Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20230305.1

### Is it regression?

No

### Expected behavior

Azure CLI 2.46.0 to be used when running an Azure CLI Task on Ubuntu 22.04.

### Actual behavior

Azure CLI 2.45.0 is used.

```
==============================================================================
Task         : Azure CLI
Description  : Run Azure CLI commands against an Azure subscription in a PowerShell Core/Shell script when running on Linux agent or PowerShell/PowerShell Core/Batch script when running on Windows agent.
Version      : 2.217.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/deploy/azure-cli
==============================================================================
/usr/bin/az --version
WARNING: You have 2 update(s) available. Consider updating your CLI installation with 'az upgrade'
azure-cli                         2.45.0 *

core                              2.45.0 *
telemetry                          1.0.8

Extensions:
azure-devops                      0.26.0

Dependencies:
msal                              1.20.0
azure-mgmt-resource             21.1.0b1

Python location '/opt/az/bin/python3'
Extensions directory '/opt/az/azcliextensions'

Python (Linux) 3.10.8 (main, Feb  3 2023, 09:38:30) [GCC 11.3.0]

Legal docs and information: aka.ms/AzureCliLegal
```

### Repro steps

1. Create a YAML pipeline with an Azure CLI task for ubuntu-22.04

**Comment by erik-bershel** [comment]
Hi @oishii-code, new AzCLI will be available with a new Ubuntu images soon.

**Comment by oishii-code** [comment]
All good now. Thanks! It's available in [20230313.1](https://github.com/actions/runner-images/blob/ubuntu22/20230313.1/images/linux/Ubuntu2204-Readme.md). Closing.
